backend/functions/ticket.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+
+function getTicket($data,$con){
+    $ticket = selectStatement($con, 'ticket', true, 'id=?', [$data['ticketId']]);
+    foreach ($ticket as $key => $value) {
+        $returnArray = [
+            'id'            => $value['id'],
+            'title'         => $value['title'],
+            'description'   => $value['description'],
+            'status'        => $value['status'],
+            'worktime'      => $value['worktime'],
+            'is_archived'   => $value['is_archived'],
+            'date_created'  => $value['date_created'] 
+        ];
+    }
+    return $returnArray;
+}
+
+function getTickets($data,$con){
+    $returnArray = [];
+
+    $tickets = selectStatement($con,'ticket',false,null,null);
+    foreach ($tickets as $key => $value) {
+
+        $companyName = "none";
+        $employeeName = "none";
+
+        $companyId = selectStatement($con, "ticket_customer",true,'`ticket_id`=?',[$value['id']]);
+
+        if(!empty($companyId)){
+            foreach ($companyId as $key2 => $value2) {
+                $company = selectStatement($con,'customer',true,'`id`=?',[$value2['customer_id']]);
+
+                foreach ($company as $key3 => $value3) {
+                    $companyName = $value3['company_name'];
+                }
+            }
+        }
+
+        $userId = selectStatement($con, 'user_ticket', true, '`ticket_id`=?', [$value['id']]);
+        
+        if(!empty($userId)){
+            foreach ($userId as $key2 => $value2) {
+                $employee = selectStatement($con, 'user', true, '`id`=?', [$value2['user_id']]);
+
+                foreach ($employee as $key3 => $value3) {
+                    $profile = selectStatement($con, 'profile',true,'`id`=?',[$value3['profile_id']]);
+
+                    foreach ($profile as $key4 => $value4) {
```

Comment:
meerdere foreach = andere naamgeving $key en $value voor betere overzicht van wat het is